### PR TITLE
Metadata glider

### PR DESCRIPTION
**Pull request by cdobs**
Completed here is the metadata review for DOSTA-M, FLORT-O, FLORD-M, FLORT-M, NUTNR-M. Changes made can be found in the notes column of the MOAS AM REVIEW tab of the CGSN Metadata Review Google Sheet (written in orange text). 

Still needed to be complete: ADCPA-M and PARADA-M.

### CODE REVIEW

File: calibration/NUTNRM/CGINS-NUTNRM-00659__20150729.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-659,CC_cal_temp,20.05,
+659,CC_cal_temp,20.05, source_file NUTNR-M_SUNA_SN_659_Calibration_Files_2015-07-29.zip > SNA0659B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00659__20150729.csv]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00660__20150729.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-660,CC_cal_temp,20.06,
+660,CC_cal_temp,20.06, source_file NUTNR-M_SUNA_SN_660_Calibration_Files_2015-07-29.zip > SNA0660B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00660__20150729.csv]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00661__20150729.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-661,CC_cal_temp,20.14,
+661,CC_cal_temp,20.14, source_file NUTNR-M_SUNA_SN_661_Calibration_Files_2015-07-29.zip > SNA0661B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00661__20150729.csv]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00662__20150723.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-662,CC_cal_temp,20.17,
+662,CC_cal_temp,20.17, source_file NUTNR-M_SUNA_SN_662_Calibration_Files_2015-07-23.zip > SNA0662A.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00662__20150723.csv]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00708__20150924.csv [hunk 5]
```diff
@@ -1,8 +1,8 @@
 serial,name,value,notes
-708,CC_cal_temp,19.93,
+708,CC_cal_temp,19.93, source_file NUTNR-M_SUNA_SN_708_Calibration_Files_2015-09-24.zip > SNA0708B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00708__20150924.csv]: Yeah, the entire file is incorrect.

File: calibration/NUTNRM/CGINS-NUTNRM-00709__20150922.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-709,CC_cal_temp,19.91,
+709,CC_cal_temp,19.91, source_file  NUTNR-M_SUNA_SN_709_Calibration_Files_2015-09-22.zip > SNA0709A.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00709__20150922.csv]: Looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00710__20150928.csv [hunk 7]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-710,CC_cal_temp,19.93,
+710,CC_cal_temp,19.93, source_file NUTNR-M_SUNA_SN_710_Calibration_Files_2015-09-28.zip > SNA0710B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00710__20150928.csv]: Fix CC_cal_temp to be 19.94

File: calibration/NUTNRM/CGINS-NUTNRM-00711__20150923.csv [hunk 8]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-711,CC_cal_temp,19.93,
+711,CC_cal_temp,19.93, source_file NUTNR-M_SUNA_SN_711_Calibration_Files_2015-09-23. zip > SNA0711A.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00711__20150923.csv]: All calibration coefficients are incorrect.

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20151013.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-729,CC_cal_temp,19.93,
+729,CC_cal_temp,19.93, source_file NUTNR-M_SUNA_SN_729_Calibration_Files_2015-10-13.zip > SNA0729B.cal
```
[the remark this answers is not in the document]
Comment (reedan88) [reply calibration/NUTNRM/CGINS-NUTNRM-00729__20151013.csv]: CC_cal_temp should be 19.98

File: calibration/FLORTM/CGINS-FLORTM-04041__20150623.csv [hunk 10]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4041,CC_angular_resolution,1.076,
+4041,CC_depolarization_ratio,0.039,
+4041,CC_measurement_wavelength,700,
+4041,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply calibration/FLORTM/CGINS-FLORTM-04041__20150623.csv]: the calibration sheets have date of 20150623.  The testing cert has 20150624 but we go by the actual cal sheets, so I think this one is ok.

File: calibration/NUTNRM/CGINS-NUTNRM-00708__20150924.csv [hunk 11]
```diff
@@ -1,8 +1,8 @@
...
..., 273.97, 274.79, 275.61, 276.44, 277.26, 278.08, 278.9, 279.72, 280.54, 281.36, 282.18, 283.01, 283.83, 284.65, 285.47, 286.29, 287.11, 287.93, 288.76, 289.58, 290.4, 291.22, 292.04, 292.86, 293.69, 294.51, 295.33, 296.15, 296.97, 297.79, 298.62, 299.44, 300.26, 301.08, 301.9, 302.72, 303.54, 304.37, 305.19, 306.01, 306.83, 307.65, 308.47, 309.29, 310.12, 310.94, 311.76, 312.58, 313.4, 314.22, 315.04, 315.86, 316.68, 317.51, 318.33, 319.15, 319.97, 320.79, 321.61, 322.43, 323.25, 324.07, 324.89, 325.71, 326.53, 327.35, 328.17, 328.99, 329.81, 330.63, 331.45, 332.27, 333.09, 333.91, 334.73, 335.55, 336.37, 337.19, 338.0, 338.82, 339.64, 340.46, 341.28, 342.1, 342.92, 343.73, 344.55, 345.37, 346.19, 347.01, 347.82, 348.64, 349.46, 350.28, 351.09, 351.91, 352.73, 353.54, 354.36, 355.18, 355.99, 356.81, 357.62, 358.44, 359.26, 360.07, 360.89, 361.7, 362.52, 363.33, 364.15, 364.96, 365.77, 366.59, 367.4, 368.22, 369.03, 369.84, 370.66, 371.47, 372.28, 373.1, 373.91, 374.72, 375.53, 376.35, 377.16, 377.97, 378.78, 379.59, 380.4, 381.21, 382.02, 382.83, 383.64, 384.45, 385.26, 386.07, 386.88, 387.69, 388.5, 389.31, 390.12, 390.93, 391.73, 392.54, 393.35, 394.16, 394.96, 395.77, 396.58]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00708__20150924.csv]: updated file looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00710__20150928.csv [hunk 12]
```diff
@@ -1,8 +1,8 @@
...
+,serial,name,value,notes
+0,710,CC_cal_temp,19.94, Source file: NUTNR-M_SUNA_SN_710_Calibration_Files_2015-09-28.zip > SNA0710B.cal
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00710__20150928.csv]: updated temp looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00711__20150923.csv [hunk 13]
```diff
@@ -1,8 +1,8 @@
...
....32, 274.14, 274.96, 275.77, 276.59, 277.4, 278.22, 279.04, 279.85, 280.67, 281.48, 282.3, 283.11, 283.93, 284.74, 285.56, 286.38, 287.19, 288.01, 288.82, 289.64, 290.45, 291.27, 292.08, 292.9, 293.71, 294.53, 295.34, 296.16, 296.97, 297.79, 298.6, 299.41, 300.23, 301.04, 301.86, 302.67, 303.49, 304.3, 305.11, 305.93, 306.74, 307.55, 308.37, 309.18, 309.99, 310.81, 311.62, 312.43, 313.25, 314.06, 314.87, 315.68, 316.5, 317.31, 318.12, 318.93, 319.74, 320.56, 321.37, 322.18, 322.99, 323.8, 324.61, 325.42, 326.24, 327.05, 327.86, 328.67, 329.48, 330.29, 331.1, 331.91, 332.72, 333.53, 334.34, 335.14, 335.95, 336.76, 337.57, 338.38, 339.19, 340.0, 340.8, 341.61, 342.42, 343.23, 344.04, 344.84, 345.65, 346.46, 347.26, 348.07, 348.88, 349.68, 350.49, 351.29, 352.1, 352.9, 353.71, 354.51, 355.32, 356.12, 356.93, 357.73, 358.53, 359.34, 360.14, 360.94, 361.75, 362.55, 363.35, 364.15, 364.96, 365.76, 366.56, 367.36, 368.16, 368.96, 369.76, 370.56, 371.36, 372.16, 372.96, 373.76, 374.56, 375.36, 376.16, 376.96, 377.76, 378.55, 379.35, 380.15, 380.95, 381.74, 382.54, 383.34, 384.13, 384.93, 385.72, 386.52, 387.31, 388.11, 388.9, 389.7, 390.49, 391.28, 392.08, 392.87, 393.66, 394.45, 395.25]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00711__20150923.csv]: updated file looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00729__20151013.csv [hunk 14]
```diff
@@ -1,8 +1,8 @@
...
+,serial,name,value,notes
+0,729,CC_cal_temp,19.98, Source file: NUTNR-M_SUNA_SN_729_Calibration_Files_2015-10-13.zip > SNA0729B.cal
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00729__20151013.csv]: updated temp looks good

**Comment by rtravis-whoi** [comment]
I just committed more changes, additions, deletions to these files based on initial review comments by AR and me.  All tracked in metadata review MOAS AM review tab.

File: calibration/DOSTAM/CGINS-DOSTAM-00019__20120504.csv [hunk 15]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+19,CC_calibration_date,1336089600000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00019__20120504.csv]: OK

File: calibration/DOSTAM/CGINS-DOSTAM-00026__20181022.csv [hunk 16]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+26,CC_calibration_date,1540166400000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00026__20181022.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00117__20160204.csv [hunk 17]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+117,CC_calibration_date,1454544000000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00117__20160204.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00117__20171015.csv [hunk 18]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+117,CC_calibration_date,1508025600000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00117__20171015.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00142__20150909.csv [hunk 19]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+142,CC_calibration_date,1441756800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00142__20150909.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00161__20160217.csv [hunk 20]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+161,CC_calibration_date,1455667200000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00161__20160217.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00161__20170531.csv [hunk 21]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+161,CC_calibration_date,1496188800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00161__20170531.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00168__20160217.csv [hunk 22]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+168,CC_calibration_date,1455667200000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00168__20160217.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00170__20170315.csv [hunk 23]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-170,CC_calibration_date,1.489536000E+12,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_170_Calibration_2017-03-15.pdf
+170,CC_calibration_date,1489536000000,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_170_Calibration_2017-03-15.pdf
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00170__20170315.csv]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00171__20130417.csv [hunk 24]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+171,CC_calibration_date,1366156800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00171__20130417.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00173__20130417.csv [hunk 25]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+173,CC_calibration_date,1366156800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (pbrickley) [inline calibration/DOSTAM/CGINS-DOSTAM-00173__20130417.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00173__20160217.csv [hunk 26]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+173,CC_calibration_date,1455667200000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00173__20160217.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00178__20130417.csv [hunk 27]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+178,CC_calibration_date,1366156800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00178__20130417.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00189__20130601.csv [hunk 28]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+189,CC_calibration_date,1370044800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00189__20130601.csv]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00190__20170315.csv [hunk 29]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+190,CC_calibration_date,1489536000000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00190__20170315.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00191__20180114.csv [hunk 30]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-191,CC_calibration_date,1.51589E+12,sensor calibration date (milliseconds since 1970-01-01)
+191,CC_calibration_date,1515888000000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00191__20180114.csv]: good

File: calibration/DOSTAM/CGINS-DOSTAM-00193__20130601.csv [hunk 31]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+193,CC_calibration_date,1370044800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00193__20130601.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00194__20130525.csv [hunk 32]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+194,CC_calibration_date,1369440000000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00194__20130525.csv]: ok

>

File: calibration/DOSTAM/CGINS-DOSTAM-00195__20130601.csv [hunk 33]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+195,CC_calibration_date,1370044800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (pbrickley) [reply calibration/DOSTAM/CGINS-DOSTAM-00195__20130601.csv]: ok

File: calibration/DOSTAM/CGINS-DOSTAM-00213__20130704.csv [hunk 34]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+213,CC_calibration_date,1365292800000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (pbrickley) [inline calibration/DOSTAM/CGINS-DOSTAM-00213__20130704.csv]: Should be: calibration/DOSTAM/CGINS-DOSTAM-00213__**20130407**.csv

Epochtime is correct.

Comment (rtravis-whoi) [reply calibration/DOSTAM/CGINS-DOSTAM-00213__20130704.csv]: good catch - I updated the date of the filename but not the epochtime.  Will fix

**Comment by rtravis-whoi** [comment]
I just updated the milliseconds in files that I previously fixed cal dates for.  They should now match up w new cal dates.  (per recent PBrickley review of a few files)

**Pull request merged**